### PR TITLE
Added bad ifName and ifAlias regex matching

### LINKED ISSUE #3422: Feature Request: Ignore interfaces based on ifName value
In the way you can:

```
$config['bad_if'][] = "voip-null";
$config['bad_iftype'][] = "voiceEncap";
```

It would be nice if you could:

```
$config['bad_ifname'][] = "port-name";
```

### PR DESCRIPTION
**Pull request by pblasquez**
Allow user to ignore interfaces based on regex matching of ifName and ifAlias values.  Fixes #3422.

### CODE REVIEW

File: includes/functions.php [hunk 1]
```diff
@@ -873,6 +873,8 @@ function is_port_valid($port, $device) {
     else {
         $valid = 1;
         $if = strtolower($port['ifDescr']);
+		$ifname = strtolower($port['ifName']);
```
Comment (laf) [inline includes/functions.php]: Can you fix the space formatting here. 4 spaces for indentation.

your other PRs had the same but I only noticed at the end :(

**Comment by pblasquez** [comment]
Ugh, sorry, using a new editor. Fixing now.

Do you want me to put in a PR to fix all of those?

**Comment by laf** [comment]
Thanks - only when you get chance.

**Comment by paulgear** [comment]
Just a note for next time: if you find yourself cutting & pasting existing code and then just changing the variable names, it probably needs to be factored out into a separate function.

**Pull request merged**